ROADMAP.md

```diff
@@ -10,75 +10,87 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
+  * Embedded mode support [#2810](https://github.com/rook/rook/issues/2810)
 * Minio
   * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+
+
+## Initial planning for Rook 1.1
+
+* Allow Ceph disk selection by full path [#2845](https://github.com/rook/rook/pull/2845)
+* Cassandra
+  * Continue to implement Cassandra design [#2294](https://github.com/rook/rook/issues/2294)
+* Ceph
+  * Remove support for Ceph Luminous
+  * Stable release of Ceph-CSI plug-in (feature parity with FlexVolume)
+  * Mon placement respects failure domains [#2603](https://github.com/rook/rook/issues/2603)
+  * User-modifiable configuration at runtime [#2470](https://github.com/rook/rook/pull/2470/files)
+  * Support modified CRUSH maps [#2514](https://github.com/rook/rook/issue/2514)
+  * OSDs
+    * Run on arbitrary PVs (local storage) as an alternative to host path [#919](https://github.com/rook/rook/issues/919)
+    * Leverage node labels for CRUSH locations [#1366](https://github.com/rook/rook/issues/1366)
+  * Orchestrate multi-site replication [#1584](https://github.com/rook/rook/issues/1584)
+  * Document a safe shutdown procedure [#2517](https://github.com/rook/rook/issues/2517)
 * NFS
   * Dynamic NFS provisioning [#2062](https://github.com/rook/rook/issues/2062)
-  * Client access control [#2283](https://github.com/rook/rook/issues/2283)
 
-## Beyond 1.0
+## Future improvements
 
-* Support for multi-networking configurations to provide more secure storage configuration (Multus?)
-* Support for more dynamic clusters such as GKE [#2107](https://github.com/rook/rook/pull/2107)
+* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
 * Integration testing improvements
-  * Incorporate new environments [#1315](https://github.com/rook/rook/issues/1315)
-  * Incorporate more architectures [#1901](https://github.com/rook/rook/issues/1901)
+  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
+  * Speed up integration tests [#1218](https://github.com/rook/rook/issues/1218)
+  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
+  * Include more architectures [#1901](https://github.com/rook/rook/issues/1901)
+    and environments [#1315](https://github.com/rook/rook/issues/1315)
+                     [#1841](https://github.com/rook/rook/issues/1841) in test pipeline
+* Support for dynamic provisioning of bucket [#1705](https://github.com/rook/rook/issues/1705)
+  and database [#1704](https://github.com/rook/rook/issues/1704) storage types
+* Support for multi-networking configurations to provide more secure storage configuration. [#2621](https://github.com/rook/rook/issues/2621)
+* Upgrade Rook to a more advanced operator/controller framework [#1981](https://github.com/rook/rook/issues/1981)
+* Support disk selection by full path [#1228](https://github.com/rook/rook/issues/1228)
```

Comment:
This is already in another section, can be removed here.

Reply:
Unless #2854 adds full path selection to the SDK, I think this issue is still valid.